app/screens/participants_list/participant_row/index.tsx

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {intlShape} from 'react-intl';
+import {Text, TouchableOpacity, View} from 'react-native';
+import {ImageSource} from 'react-native-vector-icons/Icon';
+
+import {showModal} from '@actions/navigation';
+import CompassIcon from '@components/compass_icon';
+import ProfilePicture from '@components/profile_picture';
+import type {Theme} from '@mm-redux/types/preferences';
+import {UserProfile} from '@mm-redux/types/users';
+import {displayUsername} from '@mm-redux/utils/user_utils';
+import {preventDoubleTap} from '@utils/tap';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+interface ParticipantRowProps {
+        teammateNameDisplay: string,
+        theme: Theme,
+        user: UserProfile,
+    }
+
+export default class ParticipantRow extends React.PureComponent<ParticipantRowProps> {
+    static contextTypes = {
+        intl: intlShape.isRequired,
```

Comment:
maybe in preparation for v2 that will use the most recent version of intl,  we should inject the prop instead of using the one in the context. We should also aim for a functional component instead.